tests/management/test_refresh_course_skills.py

```diff
@@ -278,10 +288,15 @@ def test_course_skill_not_saved_for_key_error(self, get_course_skills_mock, get_
     @responses.activate
     @mock.patch('taxonomy.management.commands.refresh_course_skills.get_course_metadata_provider')
     @mock.patch('taxonomy.management.commands.refresh_course_skills.utils.EMSISkillsApiClient.get_course_skills')
-    def test_course_skill_not_saved_for_type_error(self, get_course_skills_mock, get_course_provider_mock):
+    @mock.patch('taxonomy.utils.get_translated_course_description')
+    def test_course_skill_not_saved_for_type_error(self,
```

Comment:
same as above, indentation need to be updated according to https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines